test/index.js

```diff
@@ -1033,6 +1033,18 @@ describe('control', function() {
 
 });
 
+describe('number', function() {
+
+  it('returns a Just when the divisor is not 0', function() {
+    eq(S.divide(20, 5), S.Just(4));
+  });
```

Comment:
It's probably worth including a test case which demonstrates that the function does not do integer division.